corehq/apps/accounting/templates/accounting/partials/confirm_plan_summary.html

```diff
@@ -57,7 +57,16 @@ <h4>
     </h4>
   {% endif %}
 
-  {% if is_upgrade %}
+  {% if is_annual_plan %}
+    <p class="text-free">
+      {% blocktrans %}
+        By continuing you are subscribing to a Pay Annually plan and will receive an invoice via email for plan fees due.
+        In case you incur any on-demand fees for excess users or SMS, you will receive invoices for these charges on a monthly basis.
+        See <a href="https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143955299/Subscription+Management+Billing#Pay-Annually-Subscription">here</a>
+        for more information about payment, cancellation, and refunds.
+      {% endblocktrans %}
+    </p>
+  {% elif is_upgrade %}
```

Comment:
can we change `is_upgrade` to `is_monthly_upgrade` for better clarity?

Reply:
This led to some other changes to get the same logic applied, but I do think it's clearer now https://github.com/dimagi/commcare-hq/pull/36448/commits/74600ea5714212190019e37822c0b634b27f1075